lib/rucio/rse/protocols/globus.py

```diff
@@ -198,13 +198,14 @@ def delete(self, path):
             try:
                 delete_response = send_delete_task(endpoint_id=self.globus_endpoint_id, path=path, logger=self.logger)
             except TransferAPIError as err:
-                print(err)
+                self.logger(logging.WARNING, str(err))
+                raise exception.RucioException(err)
         else:
-            print('No rse attribute found for globus endpoint id.')
+            raise exception.RucioException('No rse attribute found for globus endpoint id.')
 
         if delete_response['code'] != 'Accepted':
-            print('delete_task not accepted by Globus')
-            print('delete_response: %s' % delete_response)
+            self.logger(logging.DEBUG, 'delete_response: %s' % delete_response)
```

Comment:
May I suggest deferring the log message interpolation (i.e. `s/ % /, /`)? Also below.